protocols/compound/index.json

```diff
@@ -23,5 +23,6 @@
     "claimer": "",
     "signature": ""
   },
-  "treasuryAddresses": ["0x3d9819210a31b4961b30ef54be2aed79b9c9cd3b", "0x2775b1c75658Be0F640272CCb8c72ac986009e38"]
+  "treasuryAddresses": ["0x3d9819210a31b4961b30ef54be2aed79b9c9cd3b", "0x2775b1c75658Be0F640272CCb8c72ac986009e38"],
+  "disableTreasuryNav": true
```

Comment:
Can we do `enableTreasury` instead? And only define it for protocols which don't have treasury addresses.

Reply:
That is what i did yesterday, here https://github.com/boardroom-inc/protocol-Info/pull/198.  This makes it easy to just disable for ones we don't need(Existing as well any new protocols). Most protocols will have this, if we go with `enableTreasury` then we need to add this flag for all the existing snapshot projects.

>And only define it for protocols that don't have treasury addresses. 

That is for safeAddress you mean, I can do that but I don't really see a good reason if we add safeAddress for existing protocols we have to remove it for consistency's sake, I rather leave this flag as it is.